watcher/local/watcher.go

```diff
@@ -104,3 +107,16 @@ func newCh(id, parent channel.ID, params *channel.Params) *ch {
 		parent: parent,
 	}
 }
+
+func (w *Watcher) handleEventsFromChain(eventsFromChainSub channel.AdjudicatorSubscription, eventsToClientPubSub adjudicatorPub,
+	thisCh *ch) {
+	for e := eventsFromChainSub.Next(); e != nil; e = eventsFromChainSub.Next() {
```

Comment:
`eventsFromChainSub.Err()` is unused.  
We could also change the `Next()` function to return `(event, error)`  so that correct usage is enforced.

Reply:
It was not need here, this PR is not yet handling the closing of adjudicator subscriptions.

I have added that part in a later commit (concerning the StopWatching functionality. See PR #216.